eslint.config.mjs

```diff
@@ -0,0 +1,16 @@
+import globals from 'globals';
```

Comment:
this is straight from the eslint wizard for v9, but i appreciate some comments, so we will change this up a bit now

```suggestion
import globals from 'globals';
```